games-action/prismlauncher/prismlauncher-9999.ebuild

```diff
@@ -3,7 +3,7 @@
 
 EAPI=8
 
-inherit cmake java-pkg-2 optfeature xdg
+inherit java-pkg-2 cmake optfeature toolchain-funcs xdg
```

Comment:
The change of order here is so we don't have to define `src_compile`?

I think I actually prefer it the way it was (alphabetical), it seems more readable to me since it does not rely on the implicit nature of eclass exports.